docs: drop weasel words (#22165)
renovatebot/renovate: docs/development/help-us-help-you.md

Sometimes people appear to feel frustrated that their issue or discussion has not got enough attention, and they start commenting on other issues with text such as "I'm also having problems with Gradle! See issue #xyz".
Such comments appear intended to "pressure" Renovate maintainers into taking notice, and rarely do they benefit the people subscribed to the other, mostly-unrelated issues.
Please do not do it.

> Select the "Subscribe" button on other issues to follow them.

Author: ```suggestion
Instead, use the "Subscribe" button to follow issues.
```
